services/frontend/adapters/inbound/controller.go

```diff
@@ -0,0 +1,136 @@
+package inboundAdapters
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+
+	corev1 "k8s.io/api/core/v1"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
Why is this package needed?

Reply:
> By default, plugins that use [client-go](https://godoc.org/k8s.io/client-go) cannot authenticate to Kubernetes clusters on many cloud providers. To address this, include the following import in your plugin

But I only impreted the **GCP** one, I will corect this for all of them

Reply:
I am not sure if we have to do it for all of them, as this is regarding the authentication for Claudie, not Claudie-made clusters. Therefore, as Claudie can be deployed on any cluster, we cannot be sure we will include all cloud providers. 

I am actually not sure how to approach this, should we add just major providers, or ones we use and wait for community feedback?

Reply:
This particular is Claudie to the mgmt cluster. You would prefer to remove it and wait for a direct requirement?